crane/container.go

```diff
@@ -1100,10 +1142,31 @@ func (c *container) Logs(follow bool, since string, tail string) (sources []logS
 	return
 }
 
+func (c *container) imageTag(image string, tag string) {
+	fmt.Fprintf(c.CommandsOut(), "Tagging image %s as %s...\n", image, tag)
+	args := []string{"tag", "--force", image, tag}
```

Comment:
`docker tag --force` is deprecated as of Docker 1.10, and will raise [an error in Docker 1.12](https://github.com/docker/docker/pull/18350#issuecomment-172179799)... Maybe the flag here is passed only for Docker <1.10 to allow for a smooth transition?